python/cudaq/kernel/utils.py

```diff
@@ -289,6 +289,10 @@ def pyInstanceFromName(name: str):
         return int(0)
     if name == 'float':
         return float(0.0)
+    if name in ['numpy.float32', 'np.float32']:
```

Comment:
Is `'np'` an actual namespace or a typical spelling of an alias for `'numpy'` in code? If `np` is really an alias of `numpy`, we have potential bugs here. Someone could use the `np` alias to alias some other module namespace entirely, such as
```
import fake_numpy as np
```

Reply:
Yes this is a problem with our bridge being a prototype still. There are issues like this in many places - Bruno is working on a rewrite that will support proper name resolution.